docs/index.rst

```diff
@@ -204,8 +204,6 @@ These are recipes that'll help you address specific use-cases.
 
    how-tos/*
 
-- :doc:`how-tos/casting-type-to-custom-json`
-- :doc:`how-tos/embedding-table-from-another-schema`
```

Comment:
With the removal of these the how-to's are now much more useful! :100:

Reply:
On the release page, we should document the moving of the `casting-type-to-custom-json` page to the WWTypes page.